conf/messages.en

```diff
@@ -1284,8 +1171,6 @@ declaration.summary.transport.departure.transportCode.no-code = I don’t know
 
 declaration.summary.transport.departure.meansOfTransport.header = Transport details at the border
 declaration.summary.transport.departure.meansOfTransport.header.change = Change transport details at the border
-declaration.summary.transport.departure.meansOfTransport. =
-# this is hack about incorrectness of model ^^
```

Comment:
Are you sure this is not required?

Reply:
Only if we don't have a meansOfTransport and we always do.